src/build/tenscript/library.scm

```diff
@@ -15,10 +15,31 @@
     (name "Simple" "Flagellum")
     (build
       (branch (alias Single)
-        (face (alias :next-base) (grow 3))))
+        (face (alias :next-base) (grow 20 (scale .9)))))
     (shape
       (vulcanize)
       (replace-faces)))
+  (fabric
+    (name "Simple" "Tetrapod")
```

Comment:
Random idea. Wouldn't this be nicer to read as a path? The leftmost 'labels' are conceptually similar to folders. Like this.
```suggestion
    (name "Simple/Tetrapod")
```

Reply:
Of course you'd have to figure out what to do with path components that have spaces